packages/core/src/common/quick-pick-service.ts

```diff
@@ -15,6 +15,10 @@
  ********************************************************************************/
 import { QuickOpenHideReason } from './quick-open-service';
 
+import { QuickInputTitleButton } from '../browser/quick-open/quick-title-bar';
+import { Event } from '../common/event';
+import { QuickOpenItem, QuickOpenItemOptions } from '../browser/quick-open/quick-open-model';
```

Comment:
it is not allowed to reference browser module from common, please follow https://github.com/theia-ide/theia/wiki/Code-Organization

We need to move such types to common as well in backward compatible way, i.e. via keeping files in browser and reexporting relevant types. Client code should not be affected or changed in order to verify that it is done properly.

Let me know if you need help with that. I can add a commit to refactor it.

Reply:
applied refactoring